website/sidebars.ts

```diff
@@ -555,6 +555,11 @@ const sidebars: SidebarsConfig = {
                             label: 'FedRAMP',
                             id: 'using-unleash/compliance/fedramp',
                         },
+                        {
+                            type: 'doc',
+                            label: 'SOC2',
```

Comment:
Do we just say SOC2, or SOC2 Type II?

Reply:
Let's do SOC2 Type II. that has the specific and verifiable controls that we are talking about